tests/components/lutron_caseta/test_device_trigger.py

```diff
@@ -298,6 +302,8 @@ async def test_if_fires_on_button_event_without_lip(hass, calls, device_reg):
         ATTR_DEVICE_NAME: device["Name"],
         ATTR_AREA_NAME: device.get("Area", {}).get("Name"),
         ATTR_ACTION: "press",
+        ATTR_DEVICE_ID: device_id,
+        ATTR_BUTTON_NAME: "Kitchen Pendants",
```

Comment:
This needs to be redesigned in the future since its not stable when the button is renamed